01_Exercises/Lab 4 to 6/main-family-expenses-app.py

```diff
@@ -0,0 +1,338 @@
+import removeExpenses
+import findExpenses
+import repository
+import interface
+import undoRedo
+import reports
+import os
+
+
+def collect_data():
+    """It collects the data and stores it as object"""
+    expense_date = input("\nDate (dd-mm-yy): ")
+    expense_amount = input("Amount: ")
+    expense_type = input("Type (Food, Bills, Clothes, Phone or Other): ")
+    return repository.Expense(expense_date, expense_amount, expense_type)
+
+
+def main():
+    all_expenses = []
+    undo = []
+
+    while True:
+        os.system("cls")
+        user_interface = interface.UI()
+        user_interface.main_menu()
+        options = interface.choose_option()
+
+        if options == "1":  # add expenses
+            while True:
```

Comment:
Ai un while pe fiecare branch, poti face un singur main loop in care tot verifici optiunile si iesi daca optiunea de exit a fost aleasa